ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+### Problem:
```

Comment:
I reckon you might be able to collapse this to more like 4 headings just in-case more headings make it appear daunting.

Here's a suggestion:

```md
### Describe the problem and steps to reproduce it.

### Actual results:

### Expected results: 

### Notes / URLS / Links to example add-ons or supplementary information:
```